Source/inv.cpp

```diff
@@ -908,8 +895,13 @@ void CheckInvCut(int pnum, Point cursorPosition, bool automaticMove, bool dropIt
 		}
 	}
 
-	if (dropItem) {
-		TryDropItem();
+	if (dropItem && !player.HoldItem.isEmpty()) {
+		if (IsStashOpen && AutoPlaceItemInStash(player, player.HoldItem, true)) {
+			player.HoldItem._itype = ItemType::None;

```

Comment:
They are both a reference to the same, but maybe better for consistency :)
```suggestion
	if (dropItem && !holdItem.isEmpty()) {
		if (IsStashOpen && AutoPlaceItemInStash(player, holdItem, true)) {
			holdItem._itype = ItemType::None;

```

Reply:
Fixed 🙂